src/main/java/sirius/biz/importer/format/EnumPropertyTransformer.java

```diff
@@ -32,9 +34,13 @@ protected String determineType(EnumProperty property) {
     @Override
     @SuppressWarnings("unchecked")
     protected void customizeField(EnumProperty property, FieldDefinition field) {
-        field.withCheck(new ValueInListCheck(Arrays.stream(((Class<Enum<?>>) property.getField()
-                                                                                     .getType()).getEnumConstants())
-                                                   .map(Enum::name)
-                                                   .toList()));
+        List<String> allowedValues = new ArrayList<>();
+        allowedValues.addAll(Arrays.stream(((Class<Enum<?>>) property.getField().getType()).getEnumConstants())
+                                   .map(Enum::name)
+                                   .toList());
+        allowedValues.addAll(Arrays.stream(((Class<Enum<?>>) property.getField().getType()).getEnumConstants())
+                                   .map(e -> "$" + e.getClass().getSimpleName() + "." + e.name())
+                                   .toList());
```

Comment:
Why do we have to stream twice? Just forEach instad of map and toList?
Would also remove the necessity to create two temporary lists.

Reply:
Simplified - thx :)